src/commands/sites/sites-create.js

```diff
@@ -1,58 +1,31 @@
 // @ts-check
-
-const slugify = require('@sindresorhus/slugify')
 const { InvalidArgumentError } = require('commander')
 const inquirer = require('inquirer')
 const pick = require('lodash/pick')
-const sample = require('lodash/sample')
 const prettyjson = require('prettyjson')
-const { v4: uuidv4 } = require('uuid')
 
 const { chalk, error, getRepoData, log, logJson, track, warn } = require('../../utils')
 const { configureRepo } = require('../../utils/init/config')
 const { link } = require('../link')
 
-const SITE_NAME_SUGGESTION_SUFFIX_LENGTH = 5
-
-const getSiteNameInput = async (name, user, api) => {
-  let siteSuggestion
-  if (!user) user = await api.getCurrentUser()
-
+const getSiteNameInput = async (name) => {
   if (!name) {
-    let { slug } = user
-    let suffix = ''
-
-    // If the user doesn't have a slug, we'll compute one. Because `full_name` is not guaranteed to be unique, we
-    // append a short randomly-generated ID to reduce the likelihood of a conflict.
-    if (!slug) {
-      slug = slugify(user.full_name || user.email)
-      suffix = `-${uuidv4().slice(0, SITE_NAME_SUGGESTION_SUFFIX_LENGTH)}`
-    }
-
-    const suggestions = [
-      `super-cool-site-by-${slug}${suffix}`,
-      `the-awesome-${slug}-site${suffix}`,
-      `${slug}-makes-great-sites${suffix}`,
-      `netlify-thinks-${slug}-is-great${suffix}`,
-      `the-great-${slug}-site${suffix}`,
-      `isnt-${slug}-awesome${suffix}`,
-    ]
-    siteSuggestion = sample(suggestions)
-
     const { name: nameInput } = await inquirer.prompt([
       {
         type: 'input',
         name: 'name',
         message: 'Site name (you can change it later):',
-        default: siteSuggestion,
+        default: 'Pick for me',
```

Comment:
I don't think we should be using the `default` property like this. We're assigning it a value that we know we won't use, just for the purpose of displaying something.

Instead, can we change `message` so that it reads something like `Site name (leave blank for a random name; you can change it later)`, and we assign a value if the value is empty?

If we absolutely must use the current approach, let's please move the string to a constant.

Reply:
I'll change it. I find the current approach a bit "hacky" too.